sources/platform/workflow/job/runci.md

```diff
@@ -50,6 +51,8 @@ A description of the job YML structure and the tags available is in the [jobs se
 
 * **`dependencyMode`** -- Optional. This may be set to `immediate`, `strict` or `chrono`. For detailed explanation, read about [job triggering mechanisms](/platform/workflow/overview#trigger-modes)
 
+* **`priority`** -- Optional. This may be set to any value from 0 to 10000 inclusive, with 0 being the highest priority and 10000 being the lowest priority. Jobs with higher priority will be queued before the jobs with lower priority. Default priority for all the jobs is 9999.
```

Comment:
maybe "run" instead of "queued", since the jobs will be queued but not selected to run.